R/package.R

```diff
@@ -2,7 +2,9 @@
 #'
 #' @import hyperSpec
 #' @import hySpc.testthat
-
+#'
+#' @importFrom methods is
+#' @importFrom utils maintainer
 "_PACKAGE"
```

Comment:
This file (between lines 12 and 13, i.e., between `## usethis namespace: start`  and `## usethis namespace: end`) is a good place to add `roxygen2` comment that I mentioned in some other comment:
`#' @importFrom hyperSpec  wl_convert_units`

Please, make sure that you use special comment `#'` and not `##'`

Reply:
Addressed in #3